Removing onCompleted and onError from the useQuery types (#2645)
Shopify/quilt: .changeset/breezy-dolphins-exercise.md

---
'@shopify/react-graphql': patch
---

> Fixing an issue with useQuery / Query types
> 
> `onCompleted` and `onError` callback are not supported by our custom implementation and should be exclude from the types

Reviewer: Use the active present tense:

```suggestion
Remove `onCompleted` and `onError` from the types on `useQuery`'s options as they are not implemented
```
